utils/build.py

```diff
@@ -2,6 +2,7 @@
 import os
 
 import click
+import mintotp
```

Comment:
I'm not super familiar with Python conventions, but I saw in some of your related PRs to other projects this import was done only if a TOTP secret was provided to the application, i.e. `mintotp` would never be imported if it wasn't needed. Is there any significance to that?

Reply:
All imports should be at the top of the file ([PEP8](https://peps.python.org/pep-0008/#imports)), that was me being lazy with the other projects because I don't care about code quality for them as much.

If the library wasn't installed the other scripts would break *immediately* with the missing import at the top rather than only when the secret/environment variable was defined and I was leaning on the cautious side at the time.